src/pages/admin/CreateNewMeetingPage.vue

```diff
@@ -0,0 +1,122 @@
+<script lang="ts" setup>
+import { ref } from 'vue'
+import apis, { CreateMeetingRequest } from '@/lib/apis'
+import BaseInput from '@/components/UI/BaseInput.vue'
+import BaseButton from '@/components/UI/BaseButton.vue'
+import EmoineHeader from '@/components/EmoineHeader.vue'
+import router from '@/router'
+
+//todo: 専用ヘッダー画像のアップロード
+const headerLogo = '/logo.png'
+
+const liveUrl = ref('')
+const isValidURL = ref(true)
+
+const getLiveIdFromUrl = (liveUrl: string) => {
+  const params = new URL(liveUrl).searchParams
+  if (!params.get('v')) throw new Error('Invalid URL')
+  return params.get('v')
+}
+
+const createMeeting = async () => {
+  let id = ''
+  try {
+    const videoId = getLiveIdFromUrl(liveUrl.value)
+    if (!videoId) throw new Error('Invalid URL')
+    const meetingInfo: CreateMeetingRequest = {
+      video_id: videoId,
+      description: ''
+    }
+    const res = (await apis.createMeeting(meetingInfo)).data
+    id = res.id
```

Comment:
こここうしたらif文1つ消せそう
さっきまでは`params.get('v')`を1つの変数に入れてなかったからnullかどうかの確認しても結局nullの可能性があるものとしてreturnしちゃってたけど、`videoId`に入れたことによってnullの可能性がないstringとして返すことができた
```suggestion
const getLiveIdFromUrl = (liveUrl: string) => {
  const params = new URL(liveUrl).searchParams
  const videoId = params.get('v')
  if (!videoId) throw new Error('Invalid URL')
  return videoId
}

const createMeeting = async () => {
  let id = ''
  try {
    const videoId = getLiveIdFromUrl(liveUrl.value)
    const meetingInfo: CreateMeetingRequest = {
      video_id: videoId,
      description: ''
    }
    const res = (await apis.createMeeting(meetingInfo)).data
    id = res.id
```